src/CommunityToolkit.Maui.Camera/CameraManager.macios.cs

```diff
@@ -77,24 +77,18 @@ public partial void UpdateZoom(float zoomLevel)
 		captureDevice.UnlockForConfiguration();
 	}
 
-	public async partial ValueTask UpdateCaptureResolution(Size resolution, CancellationToken token)
+	public partial ValueTask UpdateCaptureResolution(Size resolution, CancellationToken token)
 	{
-		if (captureDevice is null)
+		if (captureDevice is null || cameraView.SelectedCamera is null)
 		{
-			return;
+			return ValueTask.CompletedTask;
```

Comment:
Should we preserve the exception that was thrown when `cameraView.SelecteeCamera is null`?